common/dna_data/wc_orc_dna.txt

```diff
@@ -0,0 +1,1195 @@
+			gene_bs_nose_ridge_width={ "nose_ridge_width_pos" 73 "nose_ridge_width_pos" 186 }
+			gene_bs_nose_size={ "nose_size_neg" 253 "nose_size_neg" 50 }
+			gene_bs_nose_tip_angle={ "nose_tip_angle_pos" 139 "nose_tip_angle_pos" 234 }
+			gene_bs_nose_tip_forward={ "nose_tip_forward_neg" 255 "nose_tip_forward_neg" 254 }
+			gene_bs_nose_tip_width={ "nose_tip_width_neg" 255 "nose_tip_width_pos" 35 }
+			face_detail_cheek_def={ "cheek_def_01" 134 "cheek_def_01" 167 }
+			face_detail_cheek_fat={ "cheek_fat_01_neg" 151 "cheek_fat_02_pos" 176 }
+			face_detail_chin_cleft={ "chin_dimple" 26 "chin_dimple" 12 }
+			face_detail_chin_def={ "chin_def" 85 "chin_def" 105 }
+			face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 255 "eye_lower_lid_def" 238 }
+			face_detail_eye_socket={ "eye_socket_01" 199 "eye_socket_01" 15 }
+			face_detail_nasolabial={ "nasolabial_01" 255 "nasolabial_01" 208 }
+			face_detail_nose_ridge_def={ "nose_ridge_def_pos" 22 "nose_ridge_def_pos" 22 }
+			face_detail_nose_tip_def={ "nose_tip_def" 0 "nose_tip_def" 211 }
+			face_detail_temple_def={ "temple_def" 221 "temple_def" 221 }
+			expression_brow_wrinkles={ "brow_wrinkles_04" 251 "brow_wrinkles_04" 251 }
+			expression_eye_wrinkles={ "eye_wrinkles_03" 255 "eye_wrinkles_02" 226 }
+			expression_forehead_wrinkles={ "forehead_wrinkles_03" 255 "forehead_wrinkles_03" 197 }
+			expression_other={ "cheek_wrinkles_both_01" 255 "cheek_wrinkles_both_01" 195 }
+			complexion={ "complexion_6" 0 "complexion_3" 242 }
+			gene_height={ "normal_height" 155 "normal_height" 156 }
+			gene_bs_body_type={ "body_fat_head_fat_low" 104 "body_fat_head_fat_low" 158 }
+			gene_bs_body_shape={ "body_shape_average_clothed" 255 "body_shape_pear_half" 223 }
+			gene_bs_bust={ "bust_clothes" 138 "bust_shape_3_full" 138 }
+			gene_age={ "old_4" 255 "old_4" 141 }
+			gene_eyebrows_shape={ "no_eyebrows" 0 "avg_spacing_avg_thickness" 141 }
+			gene_eyebrows_fullness={ "no_eyebrows" 0 "layer_2_lower_thickness" 165 }
+			gene_body_hair={ "body_hair_dense_low_stubble" 0 "body_hair_sparse" 112 }
+			hairstyles={ "indian_hairstyles" 127 "orcish_hairstyles" 207 }
+			beards={ "indian_beards" 255 "orcish_beards" 105 }
+			eye_accessory={ "normal_eyes" 6 "normal_eyes" 6 }
+			teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+			eyelashes_accessory={ "no_eyelashes" 43 "no_eyelashes" 43 }
+			gene_bs_ear_lenght={ "erect_ear_lenght" 50 "erect_ear_lenght" 32 }
+			special_eyebrows={ "no_eyebrows" 255 "no_eyebrows" 6 }
+			special_eyes={ "magic_eyes" 255 "no_special_eyes" 0 }
+			gene_bs_mouth_tusks={ "mouth_fangs" 255 "mouth_fangs" 255 }
+			static_tusks={ "orcish_tusks" 81 "orcish_tusks" 81 }
+			gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 255 }
+			gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
+			clothes={ "western_bedchamber" 89 "most_clothes" 0 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+kilrogg_dna = { #by Garthor13
```

Comment:
I would make his chin is less forward.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114061316-5bada880-98a7-11eb-83ba-bf0a8b5d1a9e.png)

</details>

He lacks hairs and beards.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114061384-6f590f00-98a7-11eb-870d-803952a3a79a.png)

</details>

Reply:
Fixed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.